build/podman/pom.xml

```diff
@@ -88,4 +89,16 @@
             </build>
         </profile>
     </profiles>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>${quarkus.platform.group-id}</groupId>
+                <artifactId>quarkus-maven-plugin</artifactId>
+                <version>${quarkus.platform.version}</version>
+                <configuration>
```

Comment:
just to be clear, I think all that you need is `<quarkus.build.skip>${podman-disabled}</quarkus.build.skip>` inside `<properties` and I think all these lines are absolutely unnecessary (adding plugin) but if you are 100 % certain, np, I am not going to try it ATM

Reply:
so ideally, this PR is one-liner